ccp/src/cu/cu_prover.rs

```diff
@@ -24,14 +24,15 @@ use ccp_utils::run_utils::run_unordered;
 use cpu_utils::CPUTopology;
 
 use super::config::CUProverConfig;
+use super::proving_thread::sync::to_stats_collecting_message::ToStatsCollectingInlet;
 use super::proving_thread::ProvingThreadAsync;
 use super::proving_thread::ProvingThreadConfig;
 use super::proving_thread::ProvingThreadFacade;
 use super::proving_thread_utils::ThreadAllocator;
 use super::status::CUStatus;
 use super::status::ToCUStatus;
 use super::CUResult;
-use crate::utility_thread::message::ToUtilityInlet;
+use crate::state_operating_thread::message::ToStateOperatingInlet;
```

Comment:
I don't think it operates state, it merely saves it.

Reply:
Correct. But the struct names came from the design discussion with you, Mike and me.